config/env_production.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "production",
```

Comment:
Is this `name` collision with that defined in `env_chrome.json` any cause for concern?